ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/commands/ApplyMigrationCommand.java

```diff
@@ -312,13 +314,17 @@ private void executeCommands(
       final String previous
   ) {
     final Map<String, Object> properties = new HashMap<>();
-    final List<SqlCommand> commands = CommandParser.parse(migrationFileContent);
-    for (final SqlCommand command : commands) {
+    final List<String> commands = CommandParser.splitSql(migrationFileContent);
+
+    for (final String command : commands) {
       try {
-        executeCommand(command, ksqlClient, properties);
+        final Map<String, String> variables = ksqlClient.getVariables().entrySet()
```

Comment:
We need to reset variables between migrations files, else a DEFINE statement in one file will carry over to the next file, which we don't want.

Let's also add test coverage for this.